client/app/Dash/page.js

```diff
@@ -3,28 +3,48 @@ import React from 'react'
 import axios from 'axios'
 import './styles.css'
 import StockCard from '@/components/StockCard'
-import { useState,useEffect } from 'react'
+import { useState, useEffect } from 'react'
 import { useRouter } from 'next/navigation'
 import * as ScrollArea from '@radix-ui/react-scroll-area';
 import * as Popover from '@radix-ui/react-popover';
 import * as Toast from '@radix-ui/react-toast';
 import { MixerHorizontalIcon, Cross2Icon } from '@radix-ui/react-icons';
 import { ToastContainer, toast } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
-
-
+const availableStocks = ['Reliance', 'Tata', 'Infosys', 'HDFC', 'Wipro', 'ICICI', 'Adani', 'ITC',
```

Comment:
This doesn't contain all the stocks, try to fetch it from a seperate file and include all stocks from NSE

Reply:
yeah sure i'll look into it